attest/tpm.go

```diff
@@ -219,7 +219,7 @@ func intelEKURL(ekPub *rsa.PublicKey) string {
 	pubHash.Write(ekPub.N.Bytes())
 	pubHash.Write([]byte{0x1, 0x00, 0x01})
 
-	return intelEKCertServiceURL + base64.URLEncoding.EncodeToString(pubHash.Sum(nil))
+	return intelEKCertServiceURL + strings.ReplaceAll(base64.URLEncoding.EncodeToString(pubHash.Sum(nil)), "=", "%3D")
```

Comment:
Could this be a call to [url.QueryEscape](https://pkg.go.dev/net/url#QueryEscape) instead of directly replacing `=`?

Reply:
Fixed in f421e1de0cf130718a2810181a74d9bf1bd3ffdd. Thx! 🙂